Boa.Constrictor.UnitTests/Screenplay/Conditions/Strings/IsNullOrWhitespaceConditionTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Boa.Constrictor.Screenplay;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Boa.Constrictor.UnitTests.Screenplay
+{
+    [TestFixture]
+    public class IsNullOrWhitespaceConditionTests
```

Comment:
Thanks for writing unit tests! This is awesome.

Please update the class name to align with the new Condition name: `IsNullOrWhitespaceTests`.

Reply:
Also changed this